### ISSUE TITLE
Re-save all maps affected by data/penguins -> data/freezurbern

### ISSUE THREAD

**Issue by freezurbern** [red-eclipse/base#279]
This issue is notification that I'm re-saving maps affected by d93c67d7edb02dda0ed60ef1b11082dafdd1441e to update their .mpz.

- [x] canals
- [x] cargo
- [ ] castle
- [x] conflict
- [x] cutec
- [x] dutility
- [x] futuresport
- [x] ghost
- [x] hinder
- [x] institute
- [x] livefire
- [x] longestyard
- [x] nova
- [x] panic
- [x] pumpstation
- [x] spacetech
- [x] tribal
- [x] ubik
- [x] vault

**Comment by freezurbern** [comment]
Castle gives me:
```
2015-07-15 10:34.19 could not load side lf of sky texture penguins/wrath
2015-07-15 10:34.19 could not load side rt of sky texture penguins/wrath
2015-07-15 10:34.19 could not load side ft of sky texture penguins/wrath
2015-07-15 10:34.19 could not load side bk of sky texture penguins/wrath
2015-07-15 10:34.19 could not load side dn of sky texture penguins/wrath
2015-07-15 10:34.19 could not load side up of sky texture penguins/wrath
2015-07-15 10:34.19 warning: too many textures, [-1] textures/watern.jpg (4,4)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterdudv.jpg (4,4)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalln.jpg (4,4)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalldudv.jpg (4,4)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/watern.jpg (5,5)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterdudv.jpg (5,5)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalln.jpg (5,5)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalldudv.jpg (5,5)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/watern.jpg (6,6)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterdudv.jpg (6,6)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalln.jpg (6,6)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalldudv.jpg (6,6)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/watern.jpg (7,7)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterdudv.jpg (7,7)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalln.jpg (7,7)
2015-07-15 10:34.19 warning: too many textures, [-1] textures/waterfalldudv.jpg (7,7)
2015-07-15 10:34.19 loaded maps/castle (Castle by FaTony) v.43:226(r468) [3.5s]
```
Therefore, I will not touch it.

**Comment by freezurbern** [comment]
Fixed with c1a83af, 78656fd,fcdaa6d
